### ISSUE TITLE
Can't distribute app when adding BraintreeDropIn using SPM

### ISSUE THREAD

**Issue by JAIRMG** [braintree/braintree-ios-drop-in#441]
### Braintree Drop-in SDK Version

9.11.0

### Environment

Both

### Xcode Version

15, 15.2

### OS Version & Device

iPhone 15 iOS 17

### Integration type

Swfit Package Manager

### Development Processor

Both

### Describe the bug

We started migrating from cocoapods to SPM and we are facing an issue when we want to distribute the app

The organiser shows this error when we want to distribute the app:

No architectures in the binary. Lipo failed to detect any architectures in the bundle executable

### To reproduce

1. Import BraintreeDropIn using SPM
2. Archive  (this step is succesful)
3. Distribute: Here is failing with error:  

***No architectures in the binary. Lipo failed to detect any architectures in the bundle executable***

### Expected behavior

We would expect no errors since this is working fine if we use cocoapods

### Screenshots
![Screenshot 2024-02-21 at 15 50 34](https://github.com/braintree/braintree-ios-drop-in/assets/26467027/3af54f27-fd45-4a0e-a7f1-561b7cc04598)

![Screenshot 2024-02-21 at 15 50 03](https://github.com/braintree/braintree-ios-drop-in/assets/26467027/17ad12be-e5ab-4270-ad33-fbce60f9bdf0)

**Comment by jaxdesmarais** [comment]
Hello @JAIRMG -

Thanks for reaching out and using the Drop-in SDK. I just ran through the steps of archiving and distributing our [Demo app](https://github.com/braintree/braintree-ios-drop-in/tree/main/Demo) which uses the Drop-in as a SPM dependency. I am not seeing any issues as the distribution is working as expected.

Are you able to share a small sample app where we can reproduce this error or additional details from the logs?

**Comment by JAIRMG** [comment]
Hi @jaxdesmarais thanks for the quick response. I checked the Demo app and I see that you are pointing to master branch whereas I am using exact version 9.11.0, would that make a difference?

**Comment by jaxdesmarais** [comment]
Hey @JAIRMG - In my upload I changed from pointing to the `master` branch to point to version `9.11.0` noted in the GitHub issue.

**Comment by JAIRMG** [comment]
Hey, I found the problem. It was on my side. I had a run script that I added before to remove unused architectures and it was having problems with PPRiskMagnes and CardinalMobile. I found the log on my CI and it says:

_PPRiskMagnes.framework/PPRiskMagnes must be a fat file when the -extract option is specified
CardinalMobile must be a fat file when the -extract option is specified_

So after removing that script my upload was successful. Thanks for your help!
